documentation/release_notes.rst

```diff
@@ -8,6 +8,29 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2021.6.1
+===============
+
+Fixed Issues
+------------
+- Fixed compilation errors with C++20.
+- Fixed ``CL_OUT_OF_RESOURCES`` issue for Radix sort algorithm executed on CPU devices.
+- Fixed crashes in exclusive_scan_by_segment, inclusive_scan_by_segment, reduce_by_segment algorithms applied to
+  device-allocated unified shared memory (USM).
```

Comment:
I think wording "device-allocated unified shared memory (USM)" is confusing. I would suggest just "device-allocated memory (USM)".

Reply:
How about "applied to device-allocated USM" as was originally proposed by Alexey?

Reply:
I agree.

Reply:
Done